src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,12 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball[] balls = {new Lottery().getRandomBall(),
+                new Lottery().getRandomBall(),
+                new Lottery().getRandomBall()};
+
+        for (Ball ball : balls) {
+            System.out.println(ball);
+        }
```

Comment:
create NUMBER_OF_BALLS constant

```suggestion
        Lottery lottery = new Lottery();
        for (int i = 0; i < NUMBER_OF_BALLS; i++) {
            System.out.println(lottery.getRandomBall());
        }
```